Makefile

```diff
@@ -254,6 +254,10 @@ lint: docker-tools
 	@$(call print, "Linting source.")
 	$(DOCKER_TOOLS) golangci-lint run -v $(LINT_WORKERS)
 
+protolint:
+	@$(call print, "Linting proto files.")
+	docker run --rm --volume "$$(pwd):/workspace" --workdir /workspace yoheimuta/protolint lint .
```

Comment:
I think we should use `docker run --rm --volume "$$(pwd):/workspace" --workdir /workspace yoheimuta/protolint lint lnrpc/` instead of just `.` to limit the scope to just that directory. That should also help with the memory usage I think.

Reply:
Thanks. Yes, this makes sense!